### PR TITLE
Dz 52 complete profile

### PR DESCRIPTION
**Pull request by Stargazing-11**
## Description

This pull request implements the ability to edit user and company profiles.

### Changes Made
- Added "Edit User Profile" feature
- Added  "Edit Company Profile" functionality

### Screenshots
Screenshot from 2024-01-08 22-41-31
Screenshot from 2024-01-08 22-42-32

### CODE REVIEW

File: src/app/company-profile/service/company-profile.service.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  providedIn: 'root',
+})
+export class CompanyProfileService {
+  constructor(private http: HttpClient) {}
+
+  editCompanyProfile(
+    request: Partial<CompanyProfile>
+  ): Observable<CompanyProfile> {
+    return this.http.patch<CompanyProfile>(
+      'http://localhost:3000/companies/1',
```
Comment (destifo) [inline src/app/company-profile/service/company-profile.service.ts:16]: get the endpoint path from the api_routes.ts, do the same for all the places you provided string literal

File: src/app/user-profile/components/complete-user-profile/complete-user-profile.component.html [hunk 2]
```diff
@@ -0,0 +1,260 @@
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link
+  href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;500&display=swap"
```
Comment (destifo) [inline src/app/user-profile/components/complete-user-profile/complete-user-profile.component.html:4]: what is this?

File: src/app/user-profile/components/complete-user-profile/complete-user-profile.component.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        this.toggledProperties['isBioEditMode'] = !userProfile.bio;
+    });
+  }
+
+  ngOnDestroy(): void {
+    this.destroy$.next();
+    this.destroy$.complete();
+  }
+
+  // toggleEditMode() {
```
Comment (destifo) [inline src/app/user-profile/components/complete-user-profile/complete-user-profile.component.ts:70]: remove commented code

File: src/styles.scss [hunk 4]
```diff
@@ -35,7 +35,7 @@ $data-zone-angular-client-theme: mat.define-light-theme((
 
 /* You can add global styles to this file, and also import other style files */
 
-html, body { height: 100%; }
+// html, body { height: 100%; }
```
Comment (destifo) [inline src/styles.scss:38]: why did you comment this?

**Review by destifo — DISMISSED** [review]
address the comments

**Review by yosephhabtamu — APPROVED** [review]

**Comment by Stargazing-11** [comment]
> address the comments

Done

**Review by yosephhabtamu — APPROVED** [review]

**Pull request merged**